Auth0/CredentialsManager.swift

```diff
@@ -120,7 +121,14 @@ public struct CredentialsManager {
         self.authentication.renew(withRefreshToken: refreshToken, scope: scope).start {
             switch $0 {
             case .success(let credentials):
-                callback(nil, credentials)
+                let newCredentials = Credentials(accessToken: credentials.accessToken,
+                                                       tokenType: credentials.tokenType,
+                                                       idToken: credentials.idToken,
+                                                       refreshToken: refreshToken,
+                                                       expiresIn: credentials.expiresIn,
+                                                       scope: credentials.scope)
+                _ = self.store(credentials: newCredentials)
```

Comment:
what does the `_` mean?? I can't find it as a declared variable in the class scope.

Reply:
Is to avoiding warnings, if I set it to a variable e.g. `foo = self...` Xcode would warn that it is not used. So you use `_ = `

Reply:
Why set it anyway if it's not being used?

Reply:
You get a warning if it's not set 😄